Makefile

```diff
@@ -48,13 +67,20 @@ define get_glooe_var
 $(shell grep $(1) $(GLOOE_DIR)/build_env | cut -d '=' -f 2-)
 endef
 
-.PHONY: build-plugins
-build-plugins: $(GLOOE_DIR)/build_env $(GLOOE_DIR)/verify-plugins-linux-amd64
+.PHONY: build-plugin
+build-plugin: $(GLOOE_DIR)/build_env $(GLOOE_DIR)/verify-plugins-linux-amd64
 	docker build --no-cache \
 		--build-arg GO_BUILD_IMAGE=$(call get_glooe_var,GO_BUILD_IMAGE) \
+		--build-arg RUN_IMAGE=$(RUN_IMAGE) \
 		--build-arg GC_FLAGS=$(call get_glooe_var,GC_FLAGS) \
 		--build-arg VERIFY_SCRIPT=$(GLOOE_DIR)/verify-plugins-linux-amd64 \
-		.
+		--build-arg STORAGE_HOSTNAME=$(STORAGE_HOSTNAME) \
+		--build-arg PLUGIN_PATH=$(PLUGIN_PATH) \
+		-t $(PLUGIN_IMAGE) .
```

Comment:
i would like to see this simplified so that `get-glooe-info` and `resolve-deps` are pre-reqs to this target that run on the local machine, so that the dockerfile and makefile can be much simpler. We wouldn't need to invoke make in the dockerfile or pass tons of environment variables around as docker variables (i.e., much closer to the initial implementation)